Oryx.Cognite/src/Handler.fs

```diff
@@ -173,88 +176,90 @@ module Handler =
         req {
             let url = url +/ "byids"
             let request = ItemsWithoutCursor<Identity>(Items = ids)
-            let! ret = postV10<ItemsWithoutCursor<Identity>, ItemsWithoutCursor<'a>, 'b> request url
+            let! ret = postV10<ItemsWithoutCursor<Identity>, ItemsWithoutCursor<'a>, 'b> HttpCompletionOption.ResponseHeadersRead request url
             return ret.Items
         }
 
     let retrievePlayground<'a, 'b> (ids: Identity seq) (url: string) : HttpHandler<HttpResponseMessage, ItemsWithoutCursor<'a>, 'b> =
         req {
             let url = url +/ "byids"
             let request = ItemsWithoutCursor<Identity>(Items = ids)
-            let! ret = postPlayground<ItemsWithoutCursor<Identity>, ItemsWithoutCursor<'a>, 'b> request url
+            let! ret = postPlayground<ItemsWithoutCursor<Identity>, ItemsWithoutCursor<'a>, 'b> HttpCompletionOption.ResponseHeadersRead request url
             return ret
         }
 
     let create<'a, 'b, 'c> (content: IEnumerable<'a>) (url: string) : HttpHandler<HttpResponseMessage, IEnumerable<'b>, 'c> =
         req {
             let content' = ItemsWithoutCursor(Items=content)
-            let! ret = postV10<ItemsWithoutCursor<'a>, ItemsWithoutCursor<'b>, 'c> content' url
+            let! ret = postV10<ItemsWithoutCursor<'a>, ItemsWithoutCursor<'b>, 'c> HttpCompletionOption.ResponseContentRead content' url
             return ret.Items
         }
 
     let createPlayground<'a, 'b, 'c> (content: IEnumerable<'a>) (url: string) : HttpHandler<HttpResponseMessage, ItemsWithoutCursor<'b>, 'c> =
         req {
             let content' = ItemsWithoutCursor(Items=content)
-            let! ret = postPlayground<ItemsWithoutCursor<'a>, ItemsWithoutCursor<'b>, 'c> content' url
+            let! ret = postPlayground<ItemsWithoutCursor<'a>, ItemsWithoutCursor<'b>, 'c> HttpCompletionOption.ResponseContentRead content' url
             return ret
         }
 
     let createWithQuery<'a, 'b, 'c> (content: IEnumerable<'a>) (query: IQueryParams) (url: string) : HttpHandler<HttpResponseMessage, IEnumerable<'b>, 'c> =
         req {
             let content' = ItemsWithoutCursor(Items=content)
-            let! ret = postWithQuery<ItemsWithoutCursor<'a>, ItemsWithoutCursor<'b>, 'c> content' query url
+            let! ret = postWithQuery<ItemsWithoutCursor<'a>, ItemsWithoutCursor<'b>, 'c> HttpCompletionOption.ResponseContentRead content' query url
             return ret.Items
         }
 
     let createWithQueryEmpty<'a, 'b, 'c> (content: IEnumerable<'a>) (query: IQueryParams) (url: string) : HttpHandler<HttpResponseMessage, EmptyResponse, 'c> =
         let content' = ItemsWithoutCursor(Items=content)
-        postWithQuery<ItemsWithoutCursor<'a>, EmptyResponse, 'c> content' query url
+        postWithQuery<ItemsWithoutCursor<'a>, EmptyResponse, 'c> HttpCompletionOption.ResponseContentRead content' query url
```

Comment:
Instead of passing `completionOption` all the way to the inmost handler, let's just compose it using `withCompletion` in front of `postWithQuery` in this function. That should give the same effect and we don't have to change the signature of any the existing functions